### PR TITLE
Fix build

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: system-setup.py [hunk 1]
```diff
@@ -85,6 +85,8 @@ def common_last(self):
                      format(PYTHON=self.python, ROOT=ROOT, NOP="--nop" if self.runner.nop else ""),
                      nop=False, output=True)
         self.run("{PYTHON} {READIES}/bin/getrmpytools".format(PYTHON=self.python, READIES=READIES))
+        self.pip_uninstall("redis redis-py-cluster ramp-packer RLTest")
+        self.pip_install("git+https://github.com/RedisLabsModules/RLTest.git@v0.4.2")
```
Comment (chayim) [inline system-setup.py:89]: self.pip_install(rltest==0.4.2)

Comment (MeirShpilraien) [reply system-setup.py]: Fixed

File: plugins/jvmplugin/system-setup.py [hunk 2]
```diff
@@ -24,6 +24,8 @@ def common_last(self):
         self.install("maven")
         self.run("{PYTHON} -m pip install --upgrade pip".format(PYTHON=self.python, READIES=READIES))
         self.run("{PYTHON} {READIES}/bin/getrmpytools".format(PYTHON=self.python, READIES=READIES))
+        self.pip_uninstall("redis redis-py-cluster ramp-packer RLTest")
+        self.pip_install("git+https://github.com/RedisLabsModules/RLTest.git@v0.4.2")
```
[the remark this answers is not in the document]
Comment (MeirShpilraien) [reply plugins/jvmplugin/system-setup.py]: Fixed

**Comment by MeirShpilraien** [comment]
Handled here: https://github.com/RedisGears/RedisGears/pull/726

**Pull request closed without merging**